server/models/Notification.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: 'User'
+  },
+  type: {
+    type: String,
+    enum: ['request', 'payment', 'message'],
+    required: true
+  },
+  title: {
+    type: String,
+    default: "",
+    required: true
+  },
+  description: {
+    type: String,
+    default: "",
+        required: true
```

Comment:
Please fix the indentation here (you can setup prettier on the server side as well)

Reply:
I need help with this. I have my default code formatter set to Prettier on VScode and format on save also enabled but it still does not want to format.

Reply:
You can take a look at the prettier logs using the output window in vscode:
<img width="695" alt="vscode-output-prettier-ss" src="https://user-images.githubusercontent.com/70963670/149620219-171d9818-2fcd-470b-ab57-d8e2b889f072.png">

There might be an error in parsing the file, or maybe something else that prevents prettier from running sucessfully. Hope this helps!

Reply:
Okay I think its formatting on save but im not getting any output on my terminal like yours in the screenshot but it looks like its indenting the code properly. The indentation on this file should be correct.